src/main/java/org/jabref/logic/integrity/TitleChecker.java

```diff
@@ -48,11 +49,16 @@ public TitleChecker(BibDatabaseContext databaseContext) {
             valueOnlySpacesWithinCurlyBraces = matcher.replaceAll("");
         }
 
-        boolean hasCapitalLettersThatBibtexWillConvertToSmallerOnes = HAS_CAPITAL_LETTERS
-                .test(valueOnlySpacesWithinCurlyBraces);
-
-        if (hasCapitalLettersThatBibtexWillConvertToSmallerOnes) {
-            return Optional.of(Localization.lang("capital letters are not masked using curly brackets {}"));
+        // Delimiters are . ! ? ; :
```

Comment:
This line can be removed - I would assume that coders know that <kbd>Ctrl</kbd> + click on "DELIMITERS" leads to the definition of the variable.